### PR TITLE
Route table tweaks

### PR DESCRIPTION
**Pull request by spadgett**
Recommendations from @ncameronbritt 

- Show external link icon for route links
- "Routes To" -> "Service" in table header

### CODE REVIEW

**Comment by spadgett** [comment]
The icon can wrap by itself to the next line unfortunately. I'm not sure how to prevent this and still let long routes wrap.

openshift web console 2017-07-10 16-04-39

**Comment by ncameronbritt** [comment]
You had that done before I put up the issue!

If you append the icon without whitespace, doest it still wrap? How do we handle long URLs currently?

Also, should the icon be styled like a link? In this older screenshot from the Overview, it's not.
screen shot 2017-07-10 at 4 11 21 pm

**Comment by spadgett** [comment]
> Also, should the icon be styled like a link? In this older screenshot from the Overview, it's not.

I was trying to be consistent with other links like these. I like it better when it's not part of the link, though :/

openshift web console 2017-07-10 16-20-35

**Comment by ncameronbritt** [comment]
I think I actually like it better as part of the link. No strong preference, but we should be consistent.

Any preferences @openshift/team-ux-review ?

**Comment by spadgett** [comment]
> If you append the icon without whitespace, doest it still wrap?

Still wraps given how we break URLs.

> How do we handle long URLs currently?

We break anywhere in the value so that long URLs don't overflow the column. There might be other ways to handle it, though.

**Comment by ncameronbritt** [comment]
I know those URLs can be long... But breaking just for the icon isn't very good.

**Comment by spadgett** [comment]
For comparison, here is how the icon looks as part of the link on the overview:

openshift web console 2017-07-10 16-26-13

**Comment by spadgett** [comment]
> I know those URLs can be long... But breaking just for the icon isn't very good.

We have the same problem in other places with our warning icon unforunately.

@rhamilto @sg00dwin I wonder if

```css
  overflow: hidden;
  text-overflow: ellipsis;
```

isn't a better way of handling long unbroken strings than word break.

https://css-tricks.com/handling-long-unexpected-content-css/

**Comment by spadgett** [comment]
> I think I actually like it better as part of the link.

I think what bothers me is the icon seems too large for the rest of the text.

**Comment by beanh66** [comment]
I actually like it as part of the link too @ncameronbritt. Good point on the size though @spadgett, it wouldn't hurt to be a little smaller.

**Comment by ncameronbritt** [comment]
I think @beanh66 and I agree.

**Comment by sg00dwin** [comment]
Alternatively, since all links in that column will be external, what about denoting that by placing the icon next to the column heading.

screen shot 2017-07-11 at 10 24 27 am

**Comment by spadgett** [comment]
Icon as part of link with font-size 75%

screen shot 2017-07-11 at 10 33 23 am

screen shot 2017-07-11 at 10 33 40 am

**Comment by ncameronbritt** [comment]
I think that looks better @spadgett 

It's an interesting thought @sg00dwin. To me the icon indicates to the user what will happen when they click that link, so I'm not sure that it makes sense to include it in the table header.

**Comment by spadgett** [comment]
@ncameronbritt I looked at this again with @rhamilto, and we can't find a good way to keep the route URL wrapping but prevent the icon from ever wrapping on its own. I think we have to live with the wrapping if we want the icon.

**Comment by ncameronbritt** [comment]
I think that's ok @spadgett. Your initial comment about it gave me the impression that it happened all the time. If it's only with certain URL lengths/screen widths then it seems reasonable.

**Comment by spadgett** [comment]
Yeah, it's uncommon. It only happens if your window is exactly the wrong width.

File: app/styles/_typography.less [hunk 1]
```diff
@@ -70,6 +70,10 @@ h1 {
 
 // Misc
 // --------------------------------------------------
+a > .fa-external-link {
+  font-size: 75%;
```
Comment (jwforres) [inline app/styles/_typography.less]: how does this change affect the look of the Learn More links?

Comment (spadgett) [inline app/styles/_typography.less]: It makes them look better (IMO :)

There's one in the last screenshot above

**Comment by jwforres** [comment]
LGTM for 3.7

**Comment by spadgett** [comment]
[merge]

**Pull request merged**